packages/clerk-js/jest.config.js

```diff
@@ -4,6 +4,11 @@ const { name } = require('./package.json');
 const config = {
   displayName: name.replace('@clerk', ''),
   injectGlobals: true,
+  globals: {
+    __PKG_NAME__: '@clerk/clerk-js',
+    __PKG_VERSION__: 'test',
+    BUILD_ENABLE_NEW_COMPONENTS: '',
+  },
```

Comment:
It felt more expected to define the global variables here. Moved and added values for the `__PKG_` vars.